pulsar-client-api/pom.xml

```diff
@@ -39,5 +39,11 @@
         <version>${protobuf3.version}</version>
         <scope>provided</scope>
       </dependency>
+
+      <dependency>
```

Comment:
I don't think we should bring a 3rd party dependency at API level unless there is a strong reason we have to do so.